batect.yml

```diff
@@ -32,9 +32,6 @@ config_variables:
     default: simple
 
 .common-container-config: &common-container-config
-  run_as_current_user:
```

Comment:
Why was this removed? Running as the current user (i.e. non-root) still is a good thing, or?

Reply:
Was a test to reproduce on CI, i already check the results

Reply:
> Was a test to reproduce on CI, i already checked the results.

Just as a hint, if you'd like to check something on CI while the PR is not yet ready, you can use a draft PR. These will also trigger CI checks.